modules/introduction/pages/whats-new.adoc

```diff
@@ -1,44 +1,162 @@
+==== Improved Backup Performance 
 
-Couchbase’s unique architecture for Multi-Dimensional Scaling (MDS) allows operational and analytical queries to run concurrently without impacting one another, providing the workload isolation required for production deployments.
+Improved performance (rate of backup & restore) with backup has been the most requested factor to be adopted as an enterprise grade tool. In order to improve the performance, we made some fundamental changes to the tool. Some of these improvements include leveraging value compression on the server, replacement of storage engine, modifying the storage format, limiting the size of the backup file, metadata isolation etc.,  
+This has resulted in significant improvement in various backup functionalities such as full backup, incrementals, merge, list etc., and also the scalability for datasets in the range of single digit TBs.
+Based on our internal testing, we observe ~4x improvement compared to the previous versions of backup.
 
-To learn more about this service, or to see what's new in this production release, see xref:analytics:introduction.adoc[Couchbase Analytics].
+==== Storage
+Historically, backup was stored as one big file which at times led to the need for ~20-30% more storage compared to the size of the original dataset especially for merge operations. With the changes made to the storage engine, compression, file formats and metadata isolation, the required storage is reduced to ~40% of the original dataset.
 
-*N1QL for Analytics*
+==== Info Command (Developer Preview)
+A new  “info” command is introduced to provide a detailed listing of the backup files with information on the type of backup such as full, incremental, merge, number of views, indexes, FTS indexes, indication of backup completion etc., in addition to the repositories and their sizes.We have also added the option to output a JSON document which can be used for any automation purpose.
 
-SQL++ builds on the popularity and power of the SQL standard. It is the language used for querying schemaless, semi-structured, JSON data found in the Couchbase Data Platform.
+==== Measuring Consistency (Developer Preview)
 
-Couchbase Server 6.0 brings the first commercial implementation of the SQL++ language and uses it in the new Analytics service.
+Since Couchbase Server is a completely distributed database there is some time elapse between data distributed across nodes. With 6.5, we are providing the ability to measure this consistency for backups. To attain maximum consistency, a full backup can be run, followed by small incrementals to make up for the delta. “Disk-only” mode is used where the documents persisted to the disk are backed up which provides better consistency across vBuckets. 
```

Comment:
```suggestion
Since Couchbase Server is a completely distributed database, it takes a certain amount of time for data to be distributed across nodes. With 6.5, consistency can now be measured for backups. To attain maximum consistency, a full backup can be run, followed by small incrementals to make up for the delta. “Disk-only” mode is used where the documents persisted to the disk are backed up, which provides better consistency across vBuckets. 
```